test/unittest/mocks/mock_executor/playbook_action/mock_playbook_action_executor.go

```diff
@@ -0,0 +1,19 @@
+package mock_playbook_action_executor
+
+import (
+	"soarca/models/cacao"
+	"soarca/models/execution"
+
+	"github.com/stretchr/testify/mock"
+)
+
+type Mock_Playybook_Action_Executor struct {
```

Comment:
```suggestion
type Mock_Playbook_Action_Executor struct {
```

Typo.

Also should this be `Mock_PlaybookActionExecutor` to be more consistent with the naming of other mocks?